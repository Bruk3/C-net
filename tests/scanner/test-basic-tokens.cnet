[] {} ()
, ; '' ""

+ - * / 

= += -= == != ! < <= > >= && || .

if else for while break continue

int float char string socket struct void TCP UDP 

new delete 

return 
