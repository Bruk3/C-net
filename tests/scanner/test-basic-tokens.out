LBRACKET
RBRACKET
LBRACE
RBRACE
LPAREN
RPAREN
COMMA
SEMI
STRLIT()
PLUS
MINUS
TIMES
DIVIDE
ASSIGN
PLUSEQ
MINUSEQ
EQ
NEQ
NOT
LT
LEQ
GT
GEQ
AND
OR
DOT
IF
ELSE
FOR
WHILE
BREAK
CONTINUE
INT
FLOAT
CHAR
STRING
SOCKET
STRUCT
VOID
ID(TCP)
ID(UDP)
NEW
DELETE
RETURN
EOF
